addon/mixins/touch-action.js

```diff
@@ -45,7 +44,7 @@ export default Mixin.create({
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
     const hasTag = (typeof tagName === 'string' && tagName.length > 0) || (tagName === null && hasClick);
```

Comment:
I suspect we can just make this `const hasTag = tagName !== '' && tagName !== null`